### PR TITLE
Fix default value for sysconfig network configuration when read. (bsc#1051624)

### PR DESCRIPTION
**Pull request by teclator**

### CODE REVIEW

File: src/modules/DNS.rb [hunk 1]
```diff
@@ -282,8 +282,8 @@ def ProposeHostname
     def Read
       return true if @initialized == true
 
-      @dhcp_hostname = dhclient_set_hostname
-      @write_hostname = get_write_hostname_to_hosts
+      @dhcp_hostname = dhclient_set_hostname || false
```
Comment (mchf) [inline src/modules/DNS.rb]: np: i'm thinking of adding a note that "no" is default value in sysconfig in both cases.

File: src/modules/DNS.rb [hunk 2]
```diff
@@ -282,8 +282,8 @@ def ProposeHostname
     def Read
       return true if @initialized == true
```
Comment (mchf) [inline src/modules/DNS.rb]: you can even drop ```== true`` when tuning return value already ;-)

Comment (teclator) [inline src/modules/DNS.rb]: even :)

File: src/modules/DNS.rb (hunk 1 above)
Comment (teclator) [inline src/modules/DNS.rb]: Added a note, with bug a reference to the bug ;)

**Comment by teclator** [comment]
Thnx 👍

File: package/yast2-network.changes [hunk 3]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Aug  2 13:01:48 UTC 2017 - knut.anderssen@suse.com
+
+- bsc#1051624
+  - Fix default values for not existent sysconfig network variables 
+    when read. (bsc#1051624)
```
Comment (mchf) [inline package/yast2-network.changes]: nice to have bug reference twice ;-)

Comment (teclator) [inline package/yast2-network.changes]: xDDD

Comment (teclator) [inline package/yast2-network.changes]: Removed, added by error again after modified the changelog 👎

**Pull request merged**